### PR TITLE
#8548: switch fab to open the sidebar instead of quick bar

### LINKED ISSUE #8548: Switch FAB to open the sidebar instead of quick bar
Context
---
- Given the focus on copilots, we're now directing new users toward learning how to open the sidebar
- The updated welcome page shows users how to open the sidebar: https://www.pixiebrix.com/welcome
- The FAB still opens the quick bar. While the quick bar contains shortcuts to the open the sidebar mods, that's an extra step

Definition of Done
---
- Clicking the FAB opens the sidebar if the sidebar is currently closed
- Clicking the FAB closes the sidebar if the sidebar is currently open

Related Code
---
- https://github.com/pixiebrix/pixiebrix-extension/blob/fccd376cf7b8b15cce422ff6981840977bfd916d/src/components/floatingActions/QuickbarButton.tsx#L58-L58

### PR DESCRIPTION
**Pull request by grahamlangford**
## What does this PR do?

- Closes #8548 
- Adds Playwright tests and POM for the FAB 

## Demo

https://www.loom.com/share/0209389a01e548598daece92c04f91d3

## Future Work

- Move the FAB out of `src/components`
    - Didn't do it during this PR to preserve history/make the changes obvious

## Checklist

- [x] Add jest or playwright tests and/or storybook stories
- [x] Designate a primary reviewer @fungairino

### CODE REVIEW

File: end-to-end-tests/tests/smoke/floatingActionButton.spec.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  test("can hide the floating action button", async ({ page, extensionId }) => {
+    await page.goto("/bootstrap-5");
+
+    const floatingActionButton = new FloatingActionButton(page);
+    const actionButton = await floatingActionButton.getActionButton();
+    await expect(actionButton).toBeVisible();
+
+    await floatingActionButton.hideFloatingActionButton();
+
+    await expect(actionButton).toBeHidden();
```
Comment (fungairino) [inline end-to-end-tests/tests/smoke/floatingActionButton.spec.ts:65]: we should also verify it stays hidden if you refresh the page, right?

File: end-to-end-tests/tests/smoke/floatingActionButton.spec.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+import { ActivateModPage } from "../../pageObjects/extensionConsole/modsPage";
+import { FloatingActionButton } from "../../pageObjects/floatingActionButton";
+import { getSidebarPage, isSidebarOpen } from "../../utils";
+
+test.describe("sidebar page smoke test", () => {
+  test("can toggle the sidebar from the floating action button and view the related mod's sidebar panel", async ({
+    page,
+    extensionId,
+  }) => {
+    const modId = "@pixies/ai/writer-assist";
```
Comment (fungairino) [inline end-to-end-tests/tests/smoke/floatingActionButton.spec.ts:28]: Can we instead use a simple sidebar mod in the test team scope?

File: end-to-end-tests/tests/smoke/floatingActionButton.spec.ts (hunk 1 above)
Comment (grahamlangford) [reply end-to-end-tests/tests/smoke/floatingActionButton.spec.ts:65]: good call

File: end-to-end-tests/tests/smoke/floatingActionButton.spec.ts (hunk 2 above)
Comment (grahamlangford) [reply end-to-end-tests/tests/smoke/floatingActionButton.spec.ts:28]: Sure. I just copied from another test. But you're right. I even called out in another PR recently to prefer the @e2e scope

**Review by fungairino — APPROVED** [review]

**Pull request merged**